docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/CalculateFeatureContribution.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.ML;
+
+namespace Samples.Dynamic
+{
+    public static class CalculateFeatureContribution
+    {
+        public static void Example()
+        {
+            // Create a new context for ML.NET operations. It can be used for exception tracking and logging, 
+            // as a catalog of available operations and as the source of randomness.
+            var mlContext = new MLContext(seed: 1);
+
+            // Create a small dataset.
+            var samples = Data.GenerateData();
+
+            // Convert training data to IDataView.
+            var data = mlContext.Data.LoadFromEnumerable(samples);
+
+            // Create a pipeline to concatenate the features into a feature vector and normalize it.
+            var transformPipeline = mlContext.Transforms.Concatenate("Features", Data.FeatureColumns)
+                .Append(mlContext.Transforms.NormalizeMeanVariance("Features"));
+
+            // Fit the pipeline.
+            var transformer = transformPipeline.Fit(data);
+
+            // Transform the data.
+            var transformedData = transformer.Transform(data);
+
+            // Define a linear trainer.
+            var linearTrainer = mlContext.Regression.Trainers.Ols();
+
+            // Now we train the model and score it on the transformed data.
+            var linearModel = linearTrainer.Fit(transformedData);
+            // Print the model parameters.
+            Console.WriteLine($"Linear Model Parameters");
+            Console.WriteLine($"Bias: {linearModel.Model.Bias} Feature1: {linearModel.Model.Weights[0]} Feature2: {linearModel.Model.Weights[1]}");
+
+            // Define a feature contribution calculator for all the features, and don't normalize the contributions.
+            // These are "trivial estimators" and they don't need to fit to the data, so we can feed a subset.
+            var simpleScoredDataset = linearModel.Transform(mlContext.Data.TakeRows(transformedData, 1));
+            var linearFeatureContributionCalculator = mlContext.Transforms.CalculateFeatureContribution(linearModel, normalize: false).Fit(simpleScoredDataset);
```

Comment:
Is it possible to do `Fit(null)`? Having `simpleScoredDataset` is a bit confusing.

Reply:
It is confusing, but we cannot use `null`. We need to pass in something with a schema.